frontend/src/views/NewShoot.vue

```diff
@@ -131,6 +131,10 @@ SPDX-License-Identifier: Apache-2.0
     </div>
     <confirm-dialog ref="confirmDialog"></confirm-dialog>
   </div>
+  <v-alert class="ma-3" type="warning" v-else>
+    No supported cloud profile found.
+    Ensure that you have confiugred at least one cloud profile supported by the dashboard as well as a corresponding seed.
```

Comment:
```suggestion
    There must be at least one cloud profile supported by the dashboard as well as a corresponding seed.
```
supported by the dashboard - what does this mean?

I have adapted the message, as an enduser would not be able to ensure this

Reply:
Yes you are right. The message is intended for an operator / developer that sets up a new landscape. But you are right, an end user _could_ also see this message. I will adapt it accordingly.